kitsune/search/v2/documents.py

```diff
@@ -47,6 +48,19 @@ def update_document(cls):
     @classmethod
     def prepare(cls, instance):
         """Override super method to merge docs for KB."""
+        # Add a discard field in the document if the following conditions are met
+        # Wiki document is a redirect
+        # Wiki document is archived
+        # Wiki document is a template
+        if any(
+            [
+                instance.html.startswith(REDIRECT_HTML),
+                instance.is_archived,
+                instance.category == TEMPLATES_CATEGORY,
```

Comment:
I notice canned responses isn't excluded here (which we currently exclude alongside from templates in wiki search). Was that intentional? I can see us wanting canned responses to allow contributors to search through them to find the one they want - but I can see the same use-case for templates.

Since we don't have either feature built I guess we can remove them from ES for now and we can always start indexing them in future if we want.